packages/firebase_ml_vision/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 ## 0.4.0
 
+* **Breaking Change** Change `Rectangle<int>` to `Rect` in Text/Face/Barcode results.
```

Comment:
Version `0.4.0` has yet to be [published](https://pub.dartlang.org/packages/firebase_ml_vision#-versions-tab-), so I'm adding the change to this version.